src/controllers/artist.js

```diff
@@ -0,0 +1,99 @@
+const getDb = require("../services/db");
+
+exports.create = async (req, res) => {
+  const db = await getDb();
+  const { name, genre } = req.body;
+
+  try {
+    await db.query(`INSERT INTO Artist (name, genre) VALUES (?, ?)`, [
+      // We put a ? at any point where we want to add a variable to our
+      // statement. Then we provide those variables in an array as the second argument to db.query(). mysql2 then automatically escapes any
+      // strings that are being passed into the statement, thereby preventing any malicious SQL being executed by the database.
```

Comment:
Reminder comments like this should be removed before submitting your code for review as they dont add anything of use to the reader. If this was a particularly complex case then a comment would be useful to describe what you are doing if its not immediately obvious, but as this is a common statement it can be left out

Reply:
ok, will remove all.